c7n/resources/mq.py

```diff
@@ -37,6 +38,17 @@ class resource_type(object):
         filter_name = None
         metrics_namespace = 'AWS/AmazonMQ'
 
+    def augment(self, resources):
+        super(MessageBroker, self).augment(resources)
+        client = local_session(self.session_factory).client('mq')
+        for r in resources:
+            try:
+                tags = self.retry(client.list_tags, ResourceArn=r['BrokerArn'])['Tags']
```

Comment:
why is this being done? tags are already in describe outputs, please remove as extraneous.

Reply:
The tags are given out in the form of a dictionary. This was done to switch them to a list form

Reply:
why are tags being fetched from the api in a separate call per resource, when they are already present in the describe output?

Reply:
Understood! Thanks